src/api/pros.py

```diff
@@ -0,0 +1,118 @@
+from flask import Blueprint, jsonify, request, abort
+import uuid
+import datetime
+import json
+from utils import load_json, save_json
+
+pros = Blueprint('pros', __name__, url_prefix='/pros')
+
+
+@pros.route('/', methods=['GET'])
+def list_pros():
+    """
+    Fetches list of professionals
+    """
+    try:
+        db = load_json('./src/api/db.json')
+        if not db:
+            abort(404, description='No professionals found')
+    except (FileNotFoundError, PermissionError, json.JSONDecodeError) as e:
+        abort(500)
+
+    return jsonify(db)
+
+
+@pros.route('/', methods=['POST'])
+def create_pro():
+    """
+    Adds a new professional to directory
+    """
+    data = request.get_json()
+    if not data:
+        abort(400, description='Empty request data')
+
+    uid = str(uuid.uuid1())
+    try:
+        pros = {
+            'uid': uid,
+            'type': data['type'],
+            'firstname': data['firstname'],
+            'lastname': data['lastname'],
+            'about': data['about'],
+            'active': datetime.datetime.utcnow().timestamp(),
+            'bio': data['bio'],
+            'email': data['email'],
+            'contact': data['contact'],
+            'whatsapp': data['whatsapp'] if 'whatsapp' in data else True
+        }
+    except KeyError as e:
+        abort(400, description="%s is missing"%e)
```

Comment:
A space before and after an operator, eg: `a + b` or `"hello %s" % name`. But not around `=` for keyword arguments.

A few other places in the code as well.